### ISSUE TITLE
ValueError: 'timeout' must be a non-negative number

### ISSUE THREAD

**Issue by kanzure** [opentimestamps/opentimestamps-client#58]
```
ots: Submitting to remote calendar https://a.pool.opentimestamps.org
ots: Submitting to remote calendar https://b.pool.opentimestamps.org
ots: Submitting to remote calendar https://a.pool.eternitywall.com
Traceback (most recent call last):
  File "/home/kanzure/local/opentimestamps/opentimestamps-client/ots-git-gpg-wrapper", line 133, in <module>
    otsclient.cmds.create_timestamp(final_timestamp, args.calendar_urls, args)
  File "/home/kanzure/local/opentimestamps/opentimestamps-client/otsclient/cmds.py", line 112, in create_timestamp
    result = q.get(block=True, timeout=remaining)
  File "/usr/lib/python3.5/queue.py", line 166, in get
    raise ValueError("'timeout' must be a non-negative number")
ValueError: 'timeout' must be a non-negative number
error: gpg failed to sign the data
fatal: failed to write commit object
```

**Comment by petertodd** [comment]
This should be fixed in v0.4.0

What version is that?

**Comment by kanzure** [comment]
v0.3.4... closing.
